### PR TITLE
Animate Landing Page Lights

### PR DESCRIPTION
**Pull request by saidam90**
solves #504 

Animated lights, increased the size of lights (x3) and stretched the animation to 12s.

### CODE REVIEW

**Review by dcts — CHANGES_REQUESTED** [review]

File: tailwind.config.js
Comment (saidam90) [reply tailwind.config.js]: I fixed the typo and pushed the 12s version again, but in any case I will upload the 9s video on discord for comparison

File: src/app/page.tsx
Comment (dcts) [inline src/app/page.tsx:6]: remove outcommented line

File: src/app/page.tsx
Comment (dcts) [inline src/app/page.tsx:12]: remove empty comment

File: tailwind.config.js
Comment (dcts) [inline tailwind.config.js:23]: Why was this comment removed? 🤔

**Review by dcts — CHANGES_REQUESTED** [review]

**Comment by dcts** [comment]
@saidam90 Looks great now, happy to merge once comments get resolved! 💯

**Review by dcts — APPROVED** [review]

File: tailwind.config.js
Comment (dcts) [reply tailwind.config.js]: @saidam90 if you think this comment should be removed, please open a new PR with a reason why it should be removed :)

**Comment by dcts** [comment]
Just realized since the requested changes are so minor I can implement and commit directly.

**Comment by dcts** [comment]
@fliebenberg can you approve + merge this PR? I did, but since I made a small commit (just removing comments), I believe another maintainer review is required.

**Review by Radstakes — APPROVED** [review]

**Pull request merged**
